gemstone/src/swapper/mod.rs

```diff
@@ -81,12 +81,12 @@ impl GemSwapper {
         Self {
             rpc_provider,
             swappers: vec![
-                Box::new(universal_router::UniswapV3::new_uniswap()),
-                Box::new(universal_router::UniswapV3::new_pancakeswap()),
-                Box::new(thorchain::ThorChain::default()),
-                Box::new(jupiter::Jupiter::default()),
-                Box::new(pancakeswap_aptos::PancakeSwapAptos::default()),
-                Box::new(across::Across::default()),
+                //Box::new(universal_router::UniswapV3::new_uniswap()),
+                //Box::new(universal_router::UniswapV3::new_pancakeswap()),
+                //Box::new(thorchain::ThorChain::default()),
+                //Box::new(jupiter::Jupiter::default()),
+                //Box::new(pancakeswap_aptos::PancakeSwapAptos::default()),
+                //Box::new(across::Across::default()),
                 Box::new(stargate::Stargate::new()),
```

Comment:
can you work on provider selector next? #https://github.com/gemwalletcom/core/issues/377, with this no need to comment out next time